content/docs/tasks/security/mutual-tls/index.md

```diff
@@ -116,62 +114,70 @@ spec:
+This task shows how a server with mutual TLS enabled responses to requests that are:
 
-> In Istio 1.0, `curl` is included in proxy image, however it might be removed in future releases. In that case, you will need to install `curl` manually.
+* In plain-text
+* With TLS but without client certificate
+* With TLS with a client certificate
 
-1. Plain-text requests fail as TLS is required to talk to `httpbin`. Note the exit code is 56 (failure with receiving network data).
+To perform this task, you want to by-pass client proxy. A simplest way to do so is to issue request from `istio-proxy` container.
+
+1. Confirm that plain-text requests fail as TLS is required to talk to `httpbin` with the following command:
 
     {{< text bash >}}
     $ kubectl exec $(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name}) -c istio-proxy -- curl http://httpbin:8000/headers -o /dev/null -s -w '%{http_code}\n'
     000
     command terminated with exit code 56
     {{< /text >}}
 
-1. TLS requests without client certificate also fail, but with exit code 35 (a problem occurred somewhere in the SSL/TLS handshake).
+    > Note that the exit code is 56. The code translates to a failure to receive network data.
+
+1. Confirm TLS requests without client certificate also fail:
 
     {{< text bash >}}
     $ kubectl exec $(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name}) -c istio-proxy -- curl https://httpbin:8000/headers -o /dev/null -s -w '%{http_code}\n' -k
     000
     command terminated with exit code 35
     {{< /text >}}
 
-1. TLS request with client certificate succeed as expected:
+    > This time, exit code is 35, which corresponds to a problem occurred somewhere in the SSL/TLS handshake.
+
+1. Confirm TLS request with client certificate succeed:
 
     {{< text bash >}}
     $ kubectl exec $(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name}) -c istio-proxy -- curl https://httpbin:8000/headers -o /dev/null -s -w '%{http_code}\n' --key /etc/certs/key.pem --cert /etc/certs/cert-chain.pem --cacert /etc/certs/root-cert.pem -k
     200
     {{< /text >}}
 
 > Istio uses [Kubernetes service accounts](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/) as service identity, which
-offers stronger security than service name (refer [here](/docs/concepts/security/#identity) for more information). Thus the certificates used in Istio do
-not have service names, which is the information that `curl` needs to verify server identity. As a result, we use `curl` option `-k` to prevent the `curl`
-client from aborting when failing to find and verify the server name (i.e., `httpbin.default.svc.cluster.local`) in the certificate provided by the server.
+offers stronger security than service name (for more details, see [this](/docs/concepts/security/#istio-identity)). Thus, the certificates Istio uses do
```

Comment:
Link text should be descriptive to be able to maintain and update with ease, see my original suggestion.

Reply:
In your previous suggestion, you associate the link to term `service name`, which I feel it might be confusing. I change the text to `istio identity`, hope it's better.